api/src/test/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImplTest.java

```diff
@@ -65,11 +68,12 @@ public void setUp() throws Exception {
 	@Test
 	public void shouldFetchTranslationsForGivenLocale() throws Exception {
 		BahmniFormTranslationService bahmniFormTranslationService = new BahmniFormTranslationServiceImpl();
-		createTempFolder();
-		FormTranslation formTranslationEn = createFormTranslation("en", "1", "test_form");
-		FormTranslation formTranslationFr = createFormTranslation("fr", "1", "test_form");
-		bahmniFormTranslationService
```

Comment:
@bsneha90 any specific reason for not using  "saveFormTranslation" to save forms??

Reply:
Yes, save form translation itself is a logic, and we do not want a test to break because some other logic is broken. Test should be independent of other functionalities.

Reply:
i thought of it.. but wanted to know if it has other reasons behind it..Thanks for the confirmation !